resources/leiningen/new/compojure/project.clj

```diff
@@ -2,11 +2,11 @@
   :description "FIXME: write description"
   :url "http://example.com/FIXME"
   :min-lein-version "2.0.0"
-  :dependencies [[org.clojure/clojure "1.6.0"]
-                 [compojure "1.3.1"]
-                 [ring/ring-defaults "0.1.2"]]
-  :plugins [[lein-ring "0.8.13"]]
-  :ring {:handler {{namespace}}.handler/app}
+  :dependencies [[org.clojure/clojure "1.7.0"]
+                 [compojure "1.4.0"]
+                 [ring/ring-defaults "0.1.5"]]
+  :plugins [[lein-ring "0.9.7"]]
+  :ring {:handler {{namespace}.handler/app}}
```

Comment:
Looks like you've accidentally moved the `}` here. You'll need to put it back, or the template won't work.